emails/style.css

```diff
@@ -0,0 +1,62 @@
+.sans-serif {
```

Comment:
This doesn't look like it's actually getting loaded into the emails. Not sure how to fix that.